README.md

```diff
@@ -67,6 +67,16 @@ This connects a client that pings the `HelloWorker` and `DiceWorker` every few s
 ## Logs location
 The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.
 
+## Known Issues
+* `spatial local worker launch` is not supported.
+* `spatial project history snapshot convert` is not supported.
+* For [client workers](docs/reference/client-worker-configuration.md), it is currently not possible to specify the base command and command line arguments, with which a worker is started.
```

Comment:
This seems related to point one. Maybe say something like how you just simply need to run the base command and command line arguments yourself instead of using a json?

Reply:
Removed this point altogether.